homework_5/task3.py

```diff
@@ -0,0 +1,6 @@
+n = int(input("Enter your Number from 1 to 20: "))
+a = 0
+b = 1
+while a <= n:
+    print(a, end=" ")
+    a, b = b, a + b
```

Comment:
ლოგიკა შესასწორებელია. გიზიარებ კოდს:
n = int(input('Enter a non-negative whole number below 20: '))

if not 0 <= n < 20:
    print('Entered number does not meet the requirement.')
    exit(1)

print(0, end = ' ')

if n > 0:
    print(1, end = ' ')

if n > 1:
    index = 2
    prev1 = 1
    prev2 = 0
    while index <= n:
        index += 1
        number = prev1 + prev2
        prev2 = prev1
        prev1 = number
        print(number, end = ' ')